helm/freenas-provisioner/templates/storageclass.yaml

```diff
@@ -0,0 +1,22 @@
+---
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: freenas-nfs
```

Comment:
Name has to include Release.Name to be able to avoid collision

Reply:
I would probably make them set the storage class name manually..at a minimum let them set it then fallback to some autogenerated variant.